test/development/slow-module-detection/index.test.ts

```diff
@@ -0,0 +1,83 @@
+import { createNext, NextInstance } from 'e2e-utils'
+import { fetchViaHTTP } from 'next-test-utils'
+;(process.env.TURBOPACK ? describe.skip : describe)(
+  'Slow Module Detection',
+  () => {
+    let next: NextInstance
+
+    beforeAll(async () => {
+      next = await createNext({
+        files: {
+          'pages/index.js': `
+          import * as slowModule from '../utils/slow-module'
+          
+          export default function Page() {
+            return <div>Hello World</div>
+          }
+        `,
+          // Create a module that will be slow to compile
+          'utils/slow-module.js': `
+          // This module is intentionally made complex to trigger slow compilation
+          ${Array(2000)
+            .fill(0)
+            .map(
+              (_, i) => `
+              export const value${i} = ${JSON.stringify(
+                Array(100).fill(`some-long-string-${i}`).join('')
+              )}
+            `
+            )
+            .join('\n')}
+        `,
+          'next.config.js': `
+          module.exports = {
+            experimental: {
+              slowModuleDetectionWebpack: {
+                slowModuleThresholdMs: 50, // Lower threshold for testing
+                pathTruncationLength: 50
+              }
+            }
+          }
+        `,
+        },
+        dependencies: {
+          react: 'latest',
+          'react-dom': 'latest',
+        },
```

Comment:
We should never use npm dist-tags in tests since those could break the test at any time. There are exceptions to this rule to to pragmatic reason but the React version isn't one of them. You don't need to specify if you just use `next-test-utils`.